src/templates/sysconfig.html

```diff
@@ -619,6 +616,66 @@ <h4 class='mt-2'>Security settings</h4>
     <div class='box'>
         <h3>Identity Providers Configuration</h3>
 
+        {% if idpsArr|length > 0 %}
+            <h4>Existing Identity Providers</h4>
+            {% for idp in idpsArr %}
+                <div class='box' id='idps_div'>
```

Comment:
Shouldn't this be  `id='idps_div_{{ idp.id }}'`?

Reply:
indeed!